src/Hive.IO/Hive.IO/GhInputOutput/GhVisualizerAttributes.cs

```diff
@@ -12,15 +12,12 @@ namespace Hive.IO.GhInputOutput
 {
     public class GhVisualizerAttributes : GH_ResizableAttributes<GhVisualizer>
     {
-        // make sure we have a minimum size
-        public static float TitleBarHeight = 200f;
-        private static readonly float MinWidth = 200f;
-        private static readonly float MinHeight = TitleBarHeight + 150f;
-
         private const int Padding = 6;
+        private static readonly float MinWidth = 1144f;
+        private static readonly float MinHeight = TitleBarHeight + 580;
```

Comment:
can you half, or even third, these min width and height values, please? the component is too big in its smallest form

Reply:
OK! I've pushed some changes, please check it out on your screen for me and post the results:

minimum size:
![image](https://user-images.githubusercontent.com/2969564/95431886-34087280-094e-11eb-93db-623a913a74ed.png)

selection bar for kpi:
![image](https://user-images.githubusercontent.com/2969564/95431957-48e50600-094e-11eb-9a80-b20f25e52542.png)

Reply:
thanks, looks about the same now